src/tests/queue-test/queue-test.c

```diff
@@ -83,7 +84,8 @@ void task_1(void)
         if (status == OS_SUCCESS)
         {
             ++task_1_messages;
-            OS_printf("TASK 1: Recieved a message on the queue\n");
+            UtAssert_True(strcmp(data_received, expected) == 0, "TASK 1: data_received (%s) == expected (%s)",
+                          data_received, expected);
```

Comment:
It appears this checks for the same value in all 10 messages.  This is better than nothing but it doesn't check/verify the FIFO nature of the queue.  I'd rather see a (const) list of 10 different values that are sent in sequence and then confirmed in sequence - which will confirm that the queue implementation does indeed deliver messages in order.

Reply:
Changed it to send a sequence of 0 to 9.